judgels-backends/jerahmeel/jerahmeel-app/src/main/java/judgels/jerahmeel/stats/StatsStore.java

```diff
@@ -63,9 +63,21 @@ public StatsStore(
         this.statsUserDao = statsUserDao;
     }
 
+    private boolean isSolvableChapter(String chapterJid) {
+        return chapterProblemDao.selectCountProgrammingByChapterJid(chapterJid) > 0;
+    }
+
+    private long totalSolvableChapter(String courseJid) {
+        return courseChapterDao.selectAllByCourseJid(courseJid, SelectionOptions.DEFAULT_ALL).stream()
```

Comment:
So, this will do N DB calls, where N is total number of chapters (in production there are 13 x 2 = 26), which is not really efficient.

Can we create something like `selectCountsSolvableByChapterJids()`? Reference: https://github.com/ia-toki/judgels/blob/master/judgels-backends/jerahmeel/jerahmeel-app/src/main/java/judgels/jerahmeel/hibernate/StatsUserProblemHibernateDao.java#L87

Reply:
Used the existing selectCountProgrammingByChapterJids instead. Done.

Reply:
Oh, clever :D